api/python/quilt3/data_transfer.py

```diff
@@ -249,6 +228,34 @@ def read_file_chunks(file, chunksize=s3_transfer_config.io_chunksize):
 UPLOAD_ETAG_OPTIMIZATION_THRESHOLD = 1024
 
 
+# 8MB - same as TransferConfig().multipart_threshold - but hard-coded to guarantee it won't change.
+CHECKSUM_MULTIPART_THRESHOLD = 8 * 1024 * 1024
+
+# Maximum number of parts supported by S3
+CHECKSUM_MAX_PARTS = 10_000
+
+
+def adjust_checksum_chunksize(file_size):
```

Comment:
This might be a better name:
```suggestion
def get_checksum_chunksize(file_size):
```